### PR TITLE
setup new workflows to publish into pypi

### PR DESCRIPTION
**Pull request by sodrooome**
Related to this issue #1 

**note**
- somehow, black formatter dont support py35 version. the build still failing although already skipped if there's any failed result
- (perhaps) need some re-work in publish flow because im using default method (not using twine) to publish into pypi

### CODE REVIEW

**Comment by indrarudianto** [comment]
I consider we need to drop support for Python 3.5 as it has reached the end of its life

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+          
+      - name: Format with black
+        run: tox -e black .
```
Comment (indrarudianto) [inline .github/workflows/lint.yml:33]: No newline at the end of file

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Build binary
+        run: |
+          python -m pip install build --user
+          python -m build --sdist --wheel --outdir dist/
+
+      - name: Publish package
+        if: startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.pypi_password }}
```
Comment (indrarudianto) [inline .github/workflows/publish.yml:32]: No newline at the end of file

**Review by indrarudianto — CHANGES_REQUESTED** [review]
I think we need to just drop support for Python 3.5 as it has reached it end of life. Python 3.6 would be the lower version that still be supported, although it will be reached end of its life in 23 Dec 2021.

Thanks @sodrooome.

**Comment by sodrooome** [comment]
can you please check it again @indrarudianto ?

**Review by indrarudianto — APPROVED** [review]
All actions has passed.

**Pull request merged**